src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsRepeatableIncontentBoxadMobileWiki.java

```diff
@@ -60,15 +60,15 @@ public void adDisplayedBeforeSections() {
                          "IncontentBoxad is not displayed before section"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "IncontentBoxad is not displayed before section" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)